language-server/modules/langserver-core/src/test/resources/completion/new/statement_context/config/if_stmt_ctx_config4.json

```diff
@@ -438,20 +438,6 @@
       "insertText": "base64 `${1}`",
       "insertTextFormat": "Snippet"
     },
-    {
-      "label": "doSomeTask()",
```

Comment:
For the following cursor position,

```
public function testFunction() {
    if i<cursor>
}

function doSomeTask() {
    // todo: here goes a certain task
}
```

The `doSomeTask` function symbol is not visible